rpcs3/rpcs3qt/progress_dialog.cpp

```diff
@@ -2,10 +2,12 @@
 
 #include <QLabel>
 
-progress_dialog::progress_dialog(const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
+progress_dialog::progress_dialog(const QString& windowTitle, const QString &labelText, const QString &cancelButtonText, int minimum, int maximum, QWidget *parent, Qt::WindowFlags flags)
```

Comment:
const QString& here vs const QString & in .h.
The other args are a direct copy from Qt btw